d3a_api_client/utils.py

```diff
@@ -344,3 +341,26 @@ def get_name_from_area_name_uuid_mapping(area_name_uuid_mapping, asset_uuid):
             if area_uuid == asset_uuid:
                 return area_name
 
+
+def get_simulation_id_from_config_uuid(domain_name: str, config_uuid: str,
+                                       auth_headers: dict) -> str:
+    """Get simulation_id from configuration by performing graphql query."""
+    query = '''
+        query get_sim_id{
+            readConfiguration(uuid:"''' + config_uuid + '''"){
+                simulationResults
```

Comment:
Please try it out in the graphiql interface. The `simulationResults` contains the simlation_id as a string. This is heritage from long time ago and I understand that it can be confusing. We could rename it in the d3a-web once we get rid of d3a.io.